cc3d/core/GraphicsOffScreen/MVCDrawModelBase.py

```diff
@@ -61,56 +48,64 @@ def setParams(self):
         self.lutBlueRed.Build()
 
     def get_type_lookup_table(self, scene_metadata=None):
+        try:
+            actual_screenshot = scene_metadata['actual_screenshot']
+        except KeyError:
```

Comment:
Think this needs to handle TypeError as well, for when scene_metadata is None

Reply:
Good point

Reply:
Fixed